.github/workflows/CI-python.yml

```diff
@@ -12,17 +12,13 @@ jobs:
       node-version: 16.8
     strategy:
       matrix:
-        packageDirectory:
-          ["raiwidgets", "responsibleai", "rai_core_flask", "erroranalysis"]
+        packageDirectory: ["responsibleai", "rai_core_flask", "erroranalysis"]
         operatingSystem: [ubuntu-latest, macos-latest, windows-latest]
         pythonVersion: [3.6, 3.7, 3.8]
```

Comment:
Not for this PR, but these Python versions should probably be bumped along by .1

Reply:
I have a PR at https://github.com/microsoft/responsible-ai-toolbox/pull/1008 for adding tests for python 3.9. But some pipelines fail so I need to experiment to see which ones to exclude.